app/src/main/java/org/simple/clinic/home/HomeScreen.kt

```diff
@@ -34,6 +35,8 @@ class HomeScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context
   @Inject
   lateinit var effectHandlerFactory: HomeScreenEffectHandler.Factory
 
+  private val tabs = listOf(PATIENTS, OVERDUE, REPORTS)
```

Comment:
Is there a specific reason why we are no longer using `HomeTab.values()`?

Reply:
It creates a new array each time, plus changing the order of declarations will change the order of the tabs. I thought it would be better to make the order explicit in the screen.

Reply:
Okay 👍🏾, the reason I asked is now we have to make sure every time we added a new enum to the `HomeTab` we have to remember to add it over here as well.

Reply:
Yeah, I think that's fine because anyone will see that the tabs are not updating.

Reply:
This also gives us the flexibility to conditionally change the screens based on say, a remote config flag.

Reply:
Got it ✅